src/app/core/components/marios-card/marios-card.component.html

```diff
@@ -0,0 +1,43 @@
+<div
+  class="marios-container"
+  (click)="openDialog()"
+  (keyup)="openDialog()"
+  (mouseenter)="onMouseEnter()"
+  (mouseleave)="onMouseLeave()"
+  tabindex="0" 
+>
+  <div class="col-md-12">
+    <div class="marios-card">
+      <img src="assets/images/person.svg" alt="" />
+      <p class="marios-card-author">
+        From: {{ employee.firstName }} {{ employee.lastName }}
+      </p>
+    </div>
+  </div>
+
+  <div class="col-md-12">
+    <div class="marios-card">
+      <img src="assets/images/star.svg" alt="" />
+      <p>{{ marios.reaction | reactionMessage }}</p>
```

Comment:
Should it be p or span?

Reply:
4d6ec598a5efb33e1b76e7e56ddf83d377b7fafb